Mathlib/SetTheory/Ordinal/Exponential.lean

```diff
@@ -397,7 +397,7 @@ theorem natCast_opow (m : ℕ) : ∀ n : ℕ, ↑(m ^ n : ℕ) = (m : Ordinal) ^
   | n + 1 => by
     rw [pow_succ, natCast_mul, natCast_opow m n, Nat.cast_succ, add_one_eq_succ, opow_succ]
 
-theorem sup_opow_nat {o : Ordinal} (ho : 0 < o) : (⨆ n : ℕ, o ^ (n : Ordinal)) = o ^ ω := by
+theorem sup_opow_nat {o : Ordinal} (ho : 0 < o) : ⨆ n : ℕ, o ^ (n : Ordinal) = o ^ ω := by
```

Comment:
Unrelatedly, why is this not
```suggestion
theorem iSup_pow {o : Ordinal} (ho : 0 < o) : ⨆ n : ℕ, o ^ n = o ^ ω := by
```
?

Reply:
Probably a Lean 3 bug that was unknowingly ported to Lean 4. I'll have to check if this theorem is used anywhere in Mathlib, if it is, I'll do this fix in a separate PR.

Reply:
#16180

Reply:
I'll rename this to `iSup_pow` instead of `iSup_opow` in anticipation of that PR.